video/clip.go

```diff
@@ -16,10 +16,11 @@ type ClipStrategy struct {
 }
 
 // format time in secs to be copatible with ffmpeg's expected time syntax
-func formatTime(seconds float64) string {
```

Comment:
Is this epoch/unix time?

Reply:
It's actually the time in seconds from the *beginning* of the recording stream i.e. delta from t=0 (first segment). I'm doing a follow-on change related in this file and i'll add a comment here to make that clear.